rfcs/0093_Issue_Triage.md

```diff
@@ -0,0 +1,309 @@
+
+### Issue Filing Flow
+
+1. **Title** - Instead of including the label tags in the issue body, the user can include them in the issue title. The `triage-bot` will then parse the title as opposed to the issue body. This plan will still include a document that lists all the label tags with short descriptions. Pros: Can be more intuitive to some users. It will also lead to a shorter issue template since the “Triage” section won’t be needed anymore. Cons: Adding multiple label tags will result in a long issue title making it harder to read.
+
+![Title](assets/0089/Title.png)
+
+
+2. **Dropdown** - The dropdown approach will still only introduce one template that will include drop downs sections for each high-level component (see example below). The user will remove the tags that are irrelevant leaving behind only ones that are needed.Pros: Remove the need to refer to the documentation that lists all the label tags.Cons: The issue template will be long due to all the label tags being listed.
+
+![Dropdown](assets/0089/Dropdown.png)
+
+3. **Component Specific Issue Templates** - This approach requires that each high level component such as “frontend” or “codegen” (identified in the labels below with `<component>:` prefixes) have its own issue template. Each of these high level component templates will include a list of its sub-components in the issue body that the user can prune to only post sub-components that are relevant to their post. Pros: Remove the need for another page with all the label tags and their descriptions. Can also provide some organization structure from the start of the triage process. Cons: Limits opportunities to open issues that include multiple high-level components. Results in a long list of issue templates and can look bloated to some users.
+
+# **Future possibilities**
+
+The goal is to establish a solid issue triage workflow that will be flexible to the community’s needs as well. As TVM continues to grow, the labels used will also evolve.
+
+As a possible next step, it would be beneficial to gather a more comprehensive roster of responsible person(s) for each component and automatically assign them to issues. A [roster](https://github.com/apache/tvm/issues/10317) exists now that would need to be updated and formalized to take this effort further.
+
+It would also be helpful to change the [auto-subscription cc tracker](https://github.com/apache/tvm/issues/10317) to use this list of labels as the list of topics that can be subscribed.
+
+## New Labels
+
+This section sketches most of the labels we anticipate creating. Upon initial triage (or in response to community feedback), we may create more.
+
+* `byoc:<*codegen_backend*>` for each BYOC codegen backend in the compiler. Possible labels shown below:
+
+|Label|Scope|
+| --- | --- |
+|byoc:ethosu|python/tvm/relay/op/contrib/|
+||src/relay/backend/contrib/|
+||src/relay/op/contrib/|
+||src/runtime/contrib/|
+||src/tir/contrib/|
```

Comment:
The scope for `ethosu` mentions several high level directories here. It also appears twice in this list (same for `dnnl`).

Reply:
Ah, we should fix this. I'm going to merge this PR since the RFC vote passed, and we can make a small clarification here.